File: testdata/fixtures/scala-small/com/example/shapes/Circle.scala
package com.example.shapes

class Circle(name: String, val radius: Double) extends Base(name) with Shape {
  override def area(): Double = Circle.Pi * radius * radius
}

object Circle {
  val Pi: Double = 3.14159

  def apply(radius: Double): Circle = new Circle("unit", radius)
}

File: testdata/fixtures/kt-small/com/example/shapes/Circle.kt
package com.example.shapes

class Circle(name: String, private val radius: Double) : Base(name) {
    override fun area(): Double {
        return PI * radius * radius
    }

    companion object {
        const val PI = 3.14159

        fun unit(): Circle {
            return Circle("unit", 1.0)
        }
    }
}
